blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,245 @@
+:root {
+  --border-color: var(--c-grey-300);
+}
+
+.v2-footer {
+  background: var(--c-background-card);
+}
+
+.v2-footer :is(.v2-footer__title, p, a) {
+  font-family: var(--ff-volvo-novum);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-size: 16px;
+  letter-spacing: 0.5px;
+}
+
+.v2-footer ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.v2-footer-links-wrapper :is(.v2-footer__title, li, p, a) {
+  margin-bottom: 0;
+  color: var(--c-main-black);
+}
+
+.v2-footer-list .v2-footer__title {
+  font-family: var(--ff-volvo-novum-medium);
+  border-top: 1px solid var(--border-color);
+  cursor: pointer;
+  padding: 28px 16px;
+  display: flex;
+  align-items: center;
+}
+
+.v2-footer-list-item {
+  transition: 300ms max-height ease;
+  will-change: max-height;
+  overflow: hidden;
+  max-height: 0;
+}
+
+.v2-footer-list-item a {
+  display: flex;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+}
+
+.icon-arrow-down {
```

Comment:
This style is going to affect all the arrow downs in the code, not only to footer.

Reply:
Done